packages/test-utils/sdks/petstore/src/index.ts

```diff
@@ -3,6 +3,18 @@ import type { ConfigOptions, FetchResponse } from '@readme/api-core/types';
 import APICore from '@readme/api-core';
 import definition from '@readme/oas-examples/3.0/json/petstore.json';
 
+/**
+ * Swagger Petstore
+ *
+ * This is a sample server Petstore server.  You can find out more about Swagger at
+ * [http://swagger.io](http://swagger.io) or on [irc.freenode.net,
+ * #swagger](http://swagger.io/irc/).  For this sample, you can use the api key
+ * `special-key` to test the authorization filters.
+ *
+ * @see {@link http://swagger.io/terms/ Terms of Service}
+ * @license Apache-2.0
+ * @see {@link http://www.apache.org/licenses/LICENSE-2.0.html}
+ */
 class SDK {
```

Comment:
One annoying thing about these docblocks is that they don't really transfer to our use-case:


https://github.com/readmeio/api/assets/33762/407ad41c-b9cd-47b5-8d3f-4c02c11c3c6f

The only way to see them is if you have an original instance of the SDK class, which we don't ever expose through codegen. I still like having the docblock there are pure documentation but from an implementation point of view it kind of sucks.

Reply:
Should we add it in both spots? Or maybe. Move the SDK class into a new file and have the main index.ts file be the createSDK export?